examples_tests/50.MeshPacking/shader.frag

```diff
@@ -0,0 +1,10 @@
+#version 430 core

```

Comment:
just use the MTL shader like you would in the example about Megatextures (by having both virtual texturing and virtual geometry, you can draw the whole scene in one drawcall)